jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -170,6 +172,11 @@ org.jitsi.jigasi.transcription.SAVE_TXT=true
 org.jitsi.jigasi.transcription.SEND_TXT={{ .Env.JIGASI_TRANSCRIBER_SEND_TXT | default "false"}}
 org.jitsi.jigasi.transcription.RECORD_AUDIO={{ .Env.JIGASI_TRANSCRIBER_RECORD_AUDIO | default "false"}}
 org.jitsi.jigasi.transcription.RECORD_AUDIO_FORMAT=wav
+# non-anonymous authentication is required for transcriber
```

Comment:
These properties are already defined at line ~145: what's the difference between `JIGASI_TRANSCRIBER_USER` and `JIGASI_XMPP_USER`?

Reply:
I don't know. This may be a question to the architects of jigasi, whether a separate authentication for the transcriber is desired. I found preparations for a dedicated authentication in the code and just wanted to make the feature work.

Maybe it has to do with other jigasi features (SIP) that a separate authentication is desired?

Reply:
We don't need an extra user. JIgasi can be configured either as a trascriber or a SIP audio bridge, but not both at the same time. Thus, JIGASI_XMPP_USER (and the other ones) should be good.

Reply:
I plan to work shortly on transcribers as a first-class docker-jitsi-meet component, separated from the jigasi SIP component.  I should have PRs open by next week, in case this helps.